lib/auth/session_access_test.go

```diff
@@ -149,9 +149,50 @@ func failFilterStartTestCase(t *testing.T) startTestCase {
 	}})
 
 	return startTestCase{
-		name:        "failFilter",
-		host:        hostRole,
-		sessionKind: types.SSHSessionKind,
+		name:         "failFilter",
+		host:         []types.Role{hostRole},
+		sessionKinds: []types.SessionKind{types.SSHSessionKind},
+		participants: []SessionAccessContext{
+			{
+				Username: "participant",
+				Roles:    []types.Role{participantRole},
+				Mode:     "peer",
+			},
+			{
+				Username: "participant2",
+				Roles:    []types.Role{participantRole},
+				Mode:     "peer",
+			},
+		},
+		expected: false,
+	}
+}
+
+func failVersionCheckStartTestCase(t *testing.T) startTestCase {
```

Comment:
The unit tests are failing because this method is unused

Reply:
Thanks, that got lost in my cherry-pick, curiously VSCode didn't complain about this. Fixed.